diff -Naur mate-document-viewer-1.6.0/backend/pdf/ev-poppler.cc mate-document-viewer-1.6.0-mga/backend/pdf/ev-poppler.cc
--- mate-document-viewer-1.6.0/backend/pdf/ev-poppler.cc	2013-04-01 13:44:15.000000000 +0300
+++ mate-document-viewer-1.6.0-mga/backend/pdf/ev-poppler.cc	2013-05-12 20:54:38.118591058 +0300
@@ -640,6 +640,9 @@
 	PopplerPermissions permissions;
 	EvPage *page;
 	char *metadata;
+#ifdef HAVE_POPPLER_DOCUMENT_IS_LINEARIZED
+	gboolean linearized;
+#endif
 
 	info = g_new0 (EvDocumentInfo, 1);
 
@@ -676,7 +679,11 @@
 		      "producer", &(info->producer),
 		      "creation-date", &(info->creation_date),
 		      "mod-date", &(info->modified_date),
+#ifdef HAVE_POPPLER_DOCUMENT_IS_LINEARIZED
+		      "linearized", &linearized,
+#else
 		      "linearized", &(info->linearized),
+#endif
 		      "metadata", &metadata,
 		      NULL);
 
@@ -781,8 +788,12 @@
 		/* translators: this is the document security state */
 		info->security = g_strdup (_("No"));
 	}
+ 
+#ifdef HAVE_POPPLER_DOCUMENT_IS_LINEARIZED
+	info->linearized = linearized ? g_strdup (_("Yes")) : g_strdup (_("No"));
+#endif
 
-	return info;
+ 	return info;
 }
 
 static gboolean
diff -Naur mate-document-viewer-1.6.0/configure.ac mate-document-viewer-1.6.0-mga/configure.ac
--- mate-document-viewer-1.6.0/configure.ac	2013-04-01 13:44:15.000000000 +0300
+++ mate-document-viewer-1.6.0-mga/configure.ac	2013-05-12 20:55:41.187362407 +0300
@@ -477,6 +477,7 @@
         AC_CHECK_FUNCS(poppler_page_get_text_layout)
         AC_CHECK_FUNCS(poppler_page_get_selected_text)
         AC_CHECK_FUNCS(poppler_page_add_annot)
+        AC_CHECK_FUNCS(poppler_document_is_linearized)
         LIBS=$atril_save_LIBS
         PKG_CHECK_MODULES(CAIRO_PDF, cairo-pdf, enable_cairo_pdf=yes, enable_cairo_pdf=no)
         if test x$enable_cairo_pdf = xyes; then
